### PR TITLE
Latest data widget

### PR DESCRIPTION
**Pull request by fussel178**
### Summary 

A new widget, which displays the latest data.

### Details 

Current features:
- renders a grid based container that holds as many live widget as you want
- for every live widget you can:
  - set the event bus address
  - a title
  - the precision of the numeric value to display
  - a little status indicator whichs shows the time since the last received message
  - visual highlighting of out-of-range values
  - visual hightlighting of out-of-date values via the status indicator
  - renders per second

### Additional information 

Some sample footage:

https://user-images.githubusercontent.com/52416718/118345074-afa05280-b521-11eb-8296-8f2895f0791e.mp4

### CLA

- [x] I have signed the individual contributor's license agreement **and sent** it to the board of the WüSpace e. V. organization.

### CODE REVIEW

File: client/src/model/dashboards/widget-props/current-values-props.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { CurrentValuesWidgetProps } from '../../../widgets/current-values-widget';
+
+export const fakeProps: CurrentValuesWidgetProps = {
+	connections: [
```
Comment (pklaschka) [inline client/src/model/dashboards/widget-props/current-values-props.ts:4]: Could you please exchange the channel names with the new, "real" channel names in the mock server?

(I changed the channel names within `main` during work on this branch)

File: client/src/widgets/current-values-widget/lib/update-status-node.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		version = extractVersion(classList.value);
+		console.log(`Found spectrum StatusLight version "${version}"`);
+	}
+
+	classList.replace(
+		`_${baseClass}--${oldType}_${version}`,
+		`_${baseClass}--${newType}_${version}`
+	);
+	return newType;
+}
```
Comment (pklaschka) [inline client/src/widgets/current-values-widget/lib/update-status-node.ts:39]: Nice trick 😉

**Review by pklaschka — APPROVED** [review]
I've added one small inline suggestion (making this PR compatible with the changes introduced within `main`), but other than that, it looks great 🎉 👍

**Review by pklaschka — CHANGES_REQUESTED** [review]
That was supposed to be "Request changes". Please see my comment above 😉

File: client/src/model/dashboards/widget-props/current-values-props.ts (hunk 1 above)
Comment (fussel178) [reply client/src/model/dashboards/widget-props/current-values-props.ts:4]: Here we go.
Done in https://github.com/wuespace/telestion-project-daedalus2/pull/31/commits/a975cf6a1b015e6eaa108d81e6d046f8a97b9764

**Review by pklaschka — APPROVED** [review]
Looks good to me 😊

**Pull request merged**